### PR TITLE
added ethsafari to community events page

### PR DESCRIPTION
**Pull request by Barukimang**
Hi guys, organizer of ethsafari in Kenya here. Would like to add the community event to the community events page. Lemme know if you have questions!

### CODE REVIEW

**Comment by skylarweaver** [comment]
I support the addition of this event to the calendar! We from the Devcon team have been supporting/guiding them.

File: yarn.lock [hunk 1]
```diff
@@ -11592,7 +11592,7 @@ minimatch@^3.1.1, minimatch@^3.1.2:
   dependencies:
     brace-expansion "^1.1.7"
 
-minimist@^1.2.0, minimist@^1.2.3, minimist@^1.2.5, minimist@^1.2.6:
+minimist@^1.2.0, minimist@^1.2.3, minimist@^1.2.6:
```
Comment (minimalsm) [inline yarn.lock:11595]: Let's revert this back

```suggestion
minimist@^1.2.0, minimist@^1.2.3, minimist@^1.2.5, minimist@^1.2.6:
```

**Review by minimalsm — APPROVED** [review]
Hey @Barukimang, thanks for the PR :-) excited to see more events happening in Africa!

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @Barukimang for docs

**Comment by Barukimang** [comment]
Thanks for the review and merge @minimalsm , much appreciated! Cool, joined Discord and POAP claimed!

**Pull request merged**